gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/IndexGigawordWithElasticSearch.java

```diff
@@ -127,17 +127,24 @@ private static RestHighLevelClient buildElasticSearchClient(Parameters parameter
   private static void index(RestHighLevelClient client, Path file, String indexName) throws IOException {
     log.info("Indexing {}", file.toAbsolutePath());
 
+    Iterable<ConcatenatedGigawordDocuments.Article> concatenatedGigawordDocuments;
     // If file ends with .xml.gz, this may be the wrong version of Gigaword.
     if (file.toString().toLowerCase().endsWith(".xml.gz")) {
       log.warn("Indexing file ending with .xml.gz. This may indicate incorrect Gigaword version. "
           + "Make sure you are using the plain version LDC2011T07.");
+      concatenatedGigawordDocuments = ConcatenatedAnnotatedGigawordDocuments.fromAnnotatedGigwordGZippedFile(file);
+    }
+    else
+    {
+      concatenatedGigawordDocuments = ConcatenatedGigawordDocuments.fromGigwordGZippedFile(file);
     }
 
     // we batch the documents in groups of 100 so we can get the efficiency gains from batching without
     // making huge requests of unbounded size
-    for (List<Article> articles : partition(ConcatenatedGigawordDocuments.fromGigwordGZippedFile(file), 100)) {
+    for (List<Article> articles : partition(concatenatedGigawordDocuments, 100)) {
       final BulkRequest bulkRequest = new BulkRequest();
       for (Article article : articles) {
+        System.out.println(article.getId());
```

Comment:
This is going to lead to a lot of not very useful terminal output - we could either (a) remove it or (b) do something like print a "." every 10k document?

Reply:
Removed terminal output in newest commit.